frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -90,11 +90,11 @@ export const ParametersAndCardsContainer = styled.div`
 `;
 
 export const ParametersWidgetContainer = styled(FullWidthContainer)`
-  align-items: flex-start;
+  align-items: baseline;
```

Comment:
This helps align the parameters with the apply button.
<img width="742" alt="image" src="https://user-images.githubusercontent.com/1937582/229907891-a0650ed4-1181-45b1-b36e-8bd7f1eb3f57.png">